.markdownlinkcheck.json

```diff
@@ -1,7 +1,12 @@
 {
-    "ignorePatterns": [{
-        "pattern": "^https://calendar.google.com/calendar"
-    }],
+    "ignorePatterns": [
+        {
+            "pattern": "^https://calendar.google.com/calendar"
+        },
+        {
+            "pattern": "^https://docs.github.com/"
```

Comment:
Here is how CAPI worked around this: https://github.com/kubernetes-sigs/cluster-api/pull/6451/files#diff-a57dafecc3d9cbaf9b32b909974093b47484e25b0109096c6180ab0fe5c4e6b6R6

Sorry I meant to circle back on this and haven't had time this week... it's the github docs thing I mentioned in the office hours last week

Reply:
Much better, and this addresses what @invidian [figured out](https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/136#issuecomment-1116435670) about compression.